### ISSUE TITLE
Pre-existing models (weights)

### ISSUE THREAD

**Issue by sakshamyadav** [deepfakes/faceswap-playground#198]
MacBook Pro (13-inch, Mid 2012)
MacOS High Sierra (10.13.5)
Processor: 2.9 GHz Intel Core i7
Memory: 8 GB 1600 MHz DDR3
Graphics: Intel HD Graphics 4000 1536 MB

**Comment by sakshamyadav** [comment]
Hey guys, sorry if this has been asked already, but I couldn't find it. 

I'm currently training a Trump/Cage model with my computer (specs shown above). I don't have a GPU. After almost 12 hours of training, I'm at 830 iterations with loss A at 0.07 and loss B at 0.03. I don't mind waiting about 1 week more (if it can even be done by then). 

However, I was wondering if there is a pretrained model out there that I can use instead so I don't have to spend a long time training. Doesn't matter if its trump/cage or some other famous people, just any pretrained model at this stage will be awesome. If it doesn't exist, maybe someone could kindly share their model? 

Also, on a last note, could someone tell me if there's a way for me to speed up the training process without buying another computer (which I'll probably do later sometime)? I'm still trying to understand what GPUs are and how they work. Is it something I can buy and use with my Macbook? Thanks very much everyone! :)

**Comment by torzdf** [comment]
See the Training Data section of README.md

**Comment by sakshamyadav** [comment]
anonfile seems to be very dodgy. It installed an adware on my computer, and the download never completes due to some error.
